servers/src/main/java/tachyon/master/lineage/LineageMaster.java

```diff
@@ -266,10 +232,8 @@ public synchronized long createLineage(List<TachyonURI> inputFiles, List<Tachyon
   public synchronized boolean deleteLineage(long lineageId, boolean cascade)
       throws LineageDoesNotExistException, LineageDeletionException {
     deleteLineageInternal(lineageId, cascade);
-    DeleteLineageEntry deleteLineage = DeleteLineageEntry.newBuilder()
-        .setLineageId(lineageId)
-        .setCascade(cascade)
-        .build();
+    DeleteLineageEntry deleteLineage =
+        DeleteLineageEntry.newBuilder().setLineageId(lineageId).setCascade(cascade).build();
```

Comment:
For journal entry builders we've generally been violating the eclipse formatter to put one set per line.